var/spack/repos/builtin/packages/py-backports-functools-lru-cache/package.py

```diff
@@ -28,9 +28,10 @@
 class PyBackportsFunctoolsLruCache(PythonPackage):
     """Backport of functools.lru_cache from Python 3.3"""
 
-    homepage = "https://bitbucket.org/jaraco/backports.functools_lru_cache"
-    url      = "https://pypi.io/packages/source/b/backports.functools_lru_cache/backports.functools_lru_cache-1.0.1.zip"
+    homepage = "https://github.com/jaraco/backports.functools_lru_cache"
+    url = "https://pypi.io/packages/source/b/backports.functools_lru_cache/backports.functools_lru_cache-1.4.tar.gz"
```

Comment:
If you change from zip to tar.gz you need to make sure the old checksum is updated or the old version downloads from a different URL.